pkg/port/cli/integration.go

```diff
@@ -51,3 +65,35 @@ func (c *PortClient) GetIntegrationConfig(stateKey string) (*port.AppConfig, err
 	}
 	return pb.Integration.Config, nil
 }
+
+func (c *PortClient) DeleteIntegration(stateKey string) error {
+	resp, err := c.Client.R().
+		Delete(fmt.Sprintf("v1/integration/%s", stateKey))
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode() != 200 {
+		return fmt.Errorf("failed to delete integration, got: %s", resp.Body())
+	}
+	return nil
+}
+
+func (c *PortClient) UpdateConfig(stateKey string, config *port.AppConfig) error {
+	type Config struct {
```

Comment:
Why you need this internal struct? Just create new interface{}

Reply:
i do not and i will remove it in the following pr